fhir-server/liberty-config/configDropins/defaults/trustDefault.xml

```diff
@@ -0,0 +1,4 @@
+<server description="fhir-server">
+    <ssl id="defaultSSLConfig" trustDefaultCerts="${SEC_TLS_TRUSTDEFAULTCERTS}"/>
```

Comment:
I wonder if we need to update some docs somewhere to indicate that the server now uses the jvm's default truststore by default...

Reply:
Updated user's guide with information on this behavior.